File: tests/fixtures/counts/sample.go
package fixture

func outer() {
	label := "🦀 go work()"
	_ = label
	// go work()
	go work()
	go func() {
		go work()
		go work()
	}()
}

func boundary() {
	for i := 0; i < 10; i++ {
		go work()
	}
}

func zero() {}

type Example struct{}

func (e Example) method() {
	go work()
	go work()
}

File: tests/fixtures/counts/sample.rs
// Syntax fixtures; Badbox does not require type checking.
fn outer(value: String) {
    let label = "🦀 value.clone()";
    // value.clone();
    value.clone();
    value.clone();
    fn inner(value: String) {
        value.clone();
        value.clone();
    }
    let nested = || {
        value.clone();
        value.clone();
    };
}

fn boundary(value: String) { value.clone(); }
fn zero() {}

struct Example;
impl Example {
    fn method(&self) {
        self.clone();
        self.clone();
    }
}
